src/imports/Components/1.2/Captions.qml

```diff
@@ -124,22 +124,16 @@ ColumnLayout {
     }
 
     Label {
+        Layout.fillWidth: true
```

Comment:
Same as 1.3 Caption, is this an API break?

Reply:
No.